nexus/src/authz/omicron.polar

```diff
@@ -243,13 +243,13 @@ resource SiloUser {
 	# of the only areas of Silo configuration that Fleet Administrators have
 	# permissions on.
 	relations = { parent_silo: Silo, parent_fleet: Fleet };
-	"list_children" if "viewer" on "parent_silo";
-	"read" if "viewer" on "parent_silo";
+	"list_children" if "read" on "parent_silo";
+	"read" if "read" on "parent_silo";
```

Comment:
Checking the relevant permissions here definitely seems better than checking the role, which would give the permission indirectly. Nice.